01-node-tutorial/answers/15-create-big-file.js

```diff
@@ -1,5 +1,5 @@
 const { writeFileSync } = require('fs');
 
-for (let i = 0; i < 10000; i++) {
+for (let i = 0; i < 10000; ++i) {
```

Comment:
Is there a particular reason you changed this to a "pre-increment" symbol instead of the "post-increment" symbol?

In case you're not already familiar with the difference, you can read a bit more about it in these links:

*   [https://launchschool.com/books/javascript/read/loops_iterating](https://launchschool.com/books/javascript/read/loops_iterating) (may have to scroll down a bit to find the part; or "Control+F" to search for the term "increment"
*   [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Increment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Increment)

In the case of a for loop, it doesn't make any difference; since either way we're just adding 1 to `i`. But just wanted to call it out as something to be aware of since the difference can subtly change logic if you were for example assigning the value of the incremented variable to something else.